app/konnect/updates.md

```diff
@@ -11,6 +11,14 @@ services. [Try it today!](https://cloud.konghq.com/quick-start)
 
 ## April 2024
 
+**Manage control plane analytics data ingestion**
+: You can now opt-out from Analytics data ingestion for individual control planes. This allows you to not only manage what data is important for Kong to keep, but also your overall spending on Konnects Advanced Analytics app. <!-- if I disable this, that disables analytics for that control plane, so it won't even show up in my data in Analytics? Is the main use case here money/reducing costs? Why would someone not want to have a control plane in their analytics?-->
```

Comment:
@rosskukulinski Hi! I think you might be the one who is releasing this feature since Christian is out sick? I had a question as I was writing the changelog for the analytics data ingestion management. If you disable analytic ingestion, will that also leave any control plane data off of the Analytics page as well? Is the main use case here money/reducing costs? Why would someone not want to have a control plane in their analytics? Thanks! 

cc @sichvoge

Reply:
Hi @cloudjumpercat - primarily this is for cost-reduction purposes. We will be charging a premium for the advanced analytics capabilities in Konnect (the L1 menu: "Analytics) and customers will now have the ability to opt-out at the control plane level from ingesting data into that premium product.

Note: Pricing & packaging is changing a bit last minute and we need to delay the GA of this until May/June. Over the next few weeks @sichvoge will reach back out to you with updates!